src/lib/header-bidding/prebid/bid-config.ts

```diff
@@ -533,6 +536,20 @@ const kargoBidder: PrebidBidder = {
 	}),
 };
 
+const magniteBidder: PrebidBidder = {
+	//Rubicon is the old name for Magnite but it is still used for the integration
+	name: 'rubicon',
+	switchName: 'prebidMagnite',
+	bidParams: (
+		_slotId: string,
```

Comment:
You can lose the underscore, it just make typescript ignore unused function arguments.

Reply:
I didn't know about it. Learning something new everyday! Done 952d54fce3f22bd81cf61c266c92c35b34d5cd15